### PR TITLE
IBX-6804 - VAT rate assignment in product type

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | [IBX-6804](https://issues.ibexa.co/browse/IBX-6804) |
| Versions      | master |
| Edition       | all |

Document the updated VAT assignment procedure and accompanying changes.

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by konradoboza — APPROVED** [review]

**Review by julitafalcondusza — APPROVED** [review]

File: docs/pim/create_product_types.md [hunk 1]
```diff
@@ -38,3 +33,12 @@ Do this by clicking the **X** icon next to the group or attribute.
...
 For every attribute, you can select **Used for product variants**.
 Attributes that have this option are used to [automatically generate product variants](work_with_product_variants.md#generate-variants).
+
+6\. Below the **Attributes** section, in the **VAT rates** area, assign VAT rates that apply to all products of this type.
+
+![Assigning VAT rates](vat_rates_product_type.png "Assigning VAT rates")
+
+!!! tip
+
+    When your website supports a large number of regions, the search field helps you find the right region.
```
Comment (juskora) [inline docs/pim/create_product_types.md:43]:
```suggestion
    When your website supports a large number of regions, use the search field to find the right region.
```

**Review by juskora — APPROVED** [review]

File: docs/pim/create_product_types.md [hunk 2]
```diff
@@ -38,3 +33,12 @@ Do this by clicking the **X** icon next to the group or attribute.
...
 For every attribute, you can select **Used for product variants**.
 Attributes that have this option are used to [automatically generate product variants](work_with_product_variants.md#generate-variants).
+
+6\. Below the **Attributes** section, in the **VAT rates** area, assign VAT rates that apply to all products of this type.
+
+![Assigning VAT rates](vat_rates_product_type.png "Assigning VAT rates")
+
+!!! tip
+
+    When your website supports a large number of regions, use the search field to find the right region.
```
Comment (MagdalenaZuba) [inline docs/pim/create_product_types.md:43]:
```suggestion
    When your website supports multiple regions, use the search field to find the right one.
```

File: docs/pim/img/vat_rates_product_type.png
Comment (MagdalenaZuba) [inline docs/pim/img/vat_rates_product_type.png:1]: This screenshot is blurry.

File: docs/pim/products.md [hunk 3]
```diff
@@ -27,7 +27,11 @@ Examples: heaters, laptops, phones.
 - Virtual - non-tangible items. They can be sold individually, or as part of a product bundle.
 They don't require shipment in the online process. Examples: memberships, services, warranties. 
 
-A product type also defines the attributes that all products of this type can have.
+A product type also defines the attributes that all products of this type can have, as well as VAT rates that apply to all products of this type.
```
Comment (MagdalenaZuba) [inline docs/pim/products.md:30]:
```suggestion
A product type also defines both the attributes and VAT rates for all its products.
```

**Review by MagdalenaZuba — CHANGES_REQUESTED** [review]

**Review by MagdalenaZuba — APPROVED** [review]

**Pull request merged**